CONTRIBUTING.md

```diff
@@ -168,32 +185,72 @@ The Core Contributor group includes:
-* Offering other project support, feedback and being available to help
+* fixing, testing and triaging issues and PRs
+* offering support to the project
+* providing feedback and being available to help where needed
+* testing and reviewing pull requests
+* joining contributor meetings and supporting new contributors
 
-Varying levels of write access are made available via the project bot [Derek](https://github.com/alexellis/derek) to help regular contributors transition to GitHub Organisation Membership. Members gain access to a private Slack channel and are also featured on the Team page of the OpenFaaS website.
+> Note: An essential skill for being in a team is communication. If you cannot communicate with your team on a regular basis, then membership may not be for you and you are welcome to contribute as community.
 
-GitHub Organisation Members are expected to:
+Varying levels of write access are made available via the project bot [Derek](https://github.com/alexellis/derek) to help regular contributors transition to the Members Team.
 
+Members Team Perks:
+* access to a private Slack channel
+* profile posted on the Team page of the OpenFaaS website
+* membership of the GitHub organisations openfaas/openfaas-incubator
+
+Upon request and subject to availability:
+* 1:1 coaching & mentorship
+* help with speaking opportunities and CfP submissions
+* help with CV, resume and LinkedIn profile
+* review, and promotion of blogs and tutorials on social media
+
+The Members Team are expected to:
+
+* participate in the members channel and engage with the topics
 * participate in community Zoom calls (when possible within your timezone)
 * make regular contributions to the project codebase
-* participate in the members channel and engage with the topics
 * take an active role in the public channels: #contributors and #openfaas
 * comment on and engage with project proposals
+* attend occasional 1:1 meetings with members of the Core Team or the Project Lead
+
+This group is intended to be an active team that shares the load and collaborates together. This means engaging in topics on Slack, encouraging other teammates, sharing ideas, helping the users and raising issues with the Core Team.
+
+The Members Team will notify the Members Team of any leave of a week or more and set a status in Slack of "away".
+
+#### Changing teams
+
+Every contributor to OpenFaaS is a volunteer, including the *Project Lead* and nobody is paid to work on OpenFaaS.
+
+Motivations and life-circumstances can change over time. If this is expected to be a short-term change, then speak to the *Project Lead* about a sabbatical arrangement with perks and membership retained for that time.
+
+You may move from the Core Team to the Members Team. Please notify the *Project Lead*.
+
+If you can no-longer commit to being part of a team, then you may move to Community Contributor status and retain your access to Derek for as long as it is useful to you.
```

Comment:
`retain your access to Derek for as long as it is useful to you.`

Does this make sense?  Derek is useful for the project lead to delegate GH permissions.  How is it useful to the individual?

Reply:
I'll think about it. Thanks for your input